### ISSUE TITLE
npm test script not recognize the generated test

### ISSUE THREAD

**Issue by ZengLawrence** [Hermanya/create-react-hook#15]
npm test script did not recognize the generated test 'test.ts'.  I had to rename it to 'index.test.ts' for it to recognize.

Can it be generated as 'index.test.ts'?  I think default for jest is to look for pattern '*.test.ts'.

**Comment by Hermanya** [comment]
Right, this bug is caused by switching create react hook itself to use jest. Now I need a way to differentiate between template btests and regular tests.
